core/config/locales/en.yml

```diff
@@ -1115,6 +1115,8 @@ en:
     hints:
       spree/price:
         country: "This determines in what country the price is valid.<br/>Default: Any Country"
+        master_variant: "Changing these prices will not change variant prices below, but will be used to populate all new variants"
+        options: "These are the options used in the table below. They can be changed in the variants tab"
```

Comment:
Be more explicit with the name of the table. "These options are used to create variants in the variant table. They can be changed in the _variants tab_"